crates/bevy_render/src/mesh/mesh/mod.rs

```diff
@@ -78,50 +74,87 @@ pub struct Mesh {
-///     // and the implementation of the built-in shapes.
 /// }
 /// ```
-/// Common points of confusion:
-/// - UV maps in Bevy are "flipped", (0.0, 0.0) = Top-Left (not Bot-Left like `OpenGL`)
-/// - It is normal for multiple vertices to have the same position
-///   attribute - it's a common technique in 3D modelling for complex UV mapping or other calculations.
 ///
-/// To render correctly with `StandardMaterial` a mesh needs to have properly defined:
-/// - [`UVs`](Mesh::ATTRIBUTE_UV_0): Bevy needs to know how to map a texture onto the mesh.
-/// - [`Normals`](Mesh::ATTRIBUTE_NORMAL): Bevy needs to know how light interacts with your mesh. ([0.0, 0.0, 1.0] is very
-///              common for simple meshes because simple meshes are smooth, and they don't require complex light calculations.)
-/// - Vertex winding order -
-///              the default behavior is with `StandardMaterial.cull_mode` = Some([`Face::Front`](crate::render_resource::Face::Front)) which means
-///              that by default Bevy would *only* render the front of each triangle, and the front
-///              is the side of the triangle in which the vertices appear in a *counter-clockwise* order.
+/// You can see how it looks like [here](https://github.com/bevyengine/bevy/blob/main/assets/dovs/Mesh.png),
+/// used in a `PbrBundle` with a square bevy logo texture, with added axis, points,
+/// lines and text for clarity.
+///
+/// ## Other exampless
+///
+/// For further visualization, explanation, and examples, see the built-in Bevy examples,
+/// and the [implementation of the built-in shapes](https://github.com/bevyengine/bevy/tree/main/crates/bevy_render/src/mesh/shape).
+/// In particular, [generate_custom_mesh](https://github.com/bevyengine/bevy/blob/main/examples/3d/generate_custom_mesh.rs)
+/// teaches you to access modify a Mesh's attributes after creating it.
+///
+/// ## Common points of confusion
 ///
+/// - UV maps in Bevy start at the top-left, see [`ATTRIBUTE_UV_0`](Mesh::ATTRIBUTE_UV_0),
+/// other APIs can have other conventions, `OpenGL` starts at bottom-left.
+/// - It is possible and sometimes useful for multiple vertices to have the same
+/// [position attribute](Mesh::ATTRIBUTE_POSITION) value,
+/// it's a common technique in 3D modelling for complex UV mapping or other calculations.
+///
+/// ## Use with `StandardMaterial`
+///
+/// To render correctly with `StandardMaterial`, a mesh needs to have properly defined:
+/// - [`UVs`](Mesh::ATTRIBUTE_UV_0): Bevy needs to know how to map a texture onto the mesh
+/// (also true for `ColorMaterial`).
+/// - [`Normals`](Mesh::ATTRIBUTE_NORMAL): Bevy needs to know how light interacts with your mesh.
+/// [0.0, 0.0, 1.0] is very common for simple flat meshes on the XY plane,
+/// because simple meshes are smooth and they don't require complex light calculations.
+/// - Vertex winding order: by default, `StandardMaterial.cull_mode` is [`Some(Face::Front)`](crate::render_resource::Face),
```

Comment:
This is not correct; see https://github.com/bevyengine/bevy/blob/main/crates/bevy_pbr/src/pbr_material.rs#L191, it should be Face::Back.

The rest of the explanation in this bullet point is still correct, however.